File: Length of Last Word/length-of-last-word.py
import re


def lengthOfLastWord(s: str) -> int:
    """
    :type s: str
    :rtype: int
    """
    return len(re.findall('\w+', s).pop()) if re.match('[a-zA-Z]+', s.strip()) else 0


print(lengthOfLastWord("Hello World"))  # 5
print(lengthOfLastWord(" Hello World"))  # 5
print(lengthOfLastWord(" "))  # 0
print(lengthOfLastWord(""))  # 0

File: Length of Last Word/length-of-last-word.js
/**
 * Function find length of last word
 *
 * @param {string} s
 * @return {number}
 */
const lengthOfLastWord = s => {
  return s.search(/[a-z]/i) === -1 ? 0 : s.match(/[a-z]+/ig).pop().length;
};

console.log(lengthOfLastWord('Hello World')); // 5
console.log(lengthOfLastWord(' Hello World')); // 5
console.log(lengthOfLastWord(' ')); // 0
console.log(lengthOfLastWord('')); // 0